org.eclipse.xtext.full.releng/pom.xml

```diff
@@ -15,6 +15,7 @@
 		<module>..</module>
 		<module>../org.eclipse.xtext.nodeploy.parent</module>
 		<module>../org.eclipse.xtext.dev-bom</module>
+		<module>../org.eclipse.xtext.relengutil</module>
```

Comment:
If we don't use this new bundle during the Maven build and we don't deploy it, we could remove that from the build itself.

Reply:
I prefer it to be compiled